### DISCUSSION TITLE
more than 32 lists

### DISCUSSION THREAD

**Discussion by ShravanSunder** [community/community#35487] [Other Feature Feedback, Questions, & Ideas]
Why can't you have more than 32 lists?

**Comment by radar-blip** [comment]
I believe that is the limit for now. It may or may not increase in time.

**Comment by jfullstackdev** [comment]
why can't regular Twitter accounts edit posts ? it's not about the limitation of implementation or coding, most of the time, it's the decision of the team not do it for so many reasons

for Twitter, it's a paid feature

so, it's really a feature not a bug, isn't it ?

**Comment by Julian-Dumitrascu** [comment]
> Why can't you have more than 32 lists?

Lists of [what](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists)?

**Reply by ShravanSunder** [reply]
this section is about star or favourite lists.

**Reply by AmandaP87** [reply]
> this section is about star or favourite lists.

b866d669d664e2726a5cd9e6ad985a382e800f64

**Comment by JackXammie** [comment]
🧐

**Comment by ShravanSunder** [comment]
there is another chat about this as well:  https://github.com/orgs/community/discussions/8633

**Reply by Scotthenry2022** [reply]
Hello

**Comment by kaiser-9527** [comment]
I thought you could try this
https://hubble.monsterx.xyz/app
